### PR TITLE
vimPlugins: update all and init vim-gh-line, vis

### PR DESCRIPTION
**Pull request by kevincox**
init vim-gh-line and vis. Also update plugins.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

https://nixpk.gs/pr-tracker.html?pr=134809

### CODE REVIEW

**Comment by kevincox** [comment]
Result of `nixpkgs-review pr 134809` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>56 packages built:</summary>
  <ul>
    <li>spacevim</li>
    <li>vimPlugins.Coqtail</li>
    <li>vimPlugins.LeaderF</li>
    <li>vimPlugins.chadtree</li>
    <li>vimPlugins.coc-nvim</li>
    <li>vimPlugins.dart-vim-plugin</li>
    <li>vimPlugins.defx-nvim</li>
    <li>vimPlugins.denite (vimPlugins.denite-nvim)</li>
    <li>vimPlugins.deol-nvim</li>
    <li>vimPlugins.fugitive (vimPlugins.vim-fugitive)</li>
    <li>vimPlugins.fzf-vim</li>
    <li>vimPlugins.git-worktree-nvim</li>
    <li>vimPlugins.gitsigns-nvim</li>
    <li>vimPlugins.glow-nvim</li>
    <li>vimPlugins.gruvbox-community</li>
    <li>vimPlugins.gruvbox-flat-nvim</li>
    <li>vimPlugins.gruvbox-nvim</li>
    <li>vimPlugins.lh-brackets</li>
    <li>vimPlugins.lh-vim-lib</li>
    <li>vimPlugins.lightspeed-nvim</li>
    <li>vimPlugins.lsp_signature-nvim</li>
    <li>vimPlugins.lspkind-nvim</li>
    <li>vimPlugins.luasnip</li>
    <li>vimPlugins.neco-vim</li>
    <li>vimPlugins.neogit</li>
    <li>vimPlugins.nerdtree-git-plugin</li>
    <li>vimPlugins.null-ls-nvim</li>
    <li>vimPlugins.nvim-cmp</li>
    <li>vimPlugins.nvim-compe</li>
    <li>vimPlugins.nvim-dap</li>
    <li>vimPlugins.nvim-hlslens</li>
    <li>vimPlugins.nvim-lspconfig</li>
    <li>vimPlugins.nvim-ts-context-commentstring</li>
    <li>vimPlugins.nvim-ts-rainbow</li>
    <li>vimPlugins.refactoring-nvim</li>
    <li>vimPlugins.sql-nvim</li>
    <li>vimPlugins.symbols-outline-nvim</li>
    <li>vimPlugins.telescope-frecency-nvim</li>
    <li>vimPlugins.telescope-fzf-native-nvim</li>
    <li>vimPlugins.telescope-fzf-writer-nvim</li>
    <li>vimPlugins.telescope-fzy-native-nvim</li>
    <li>vimPlugins.telescope-nvim</li>
    <li>vimPlugins.telescope-symbols-nvim</li>
    <li>vimPlugins.telescope-z-nvim</li>
    <li>vimPlugins.vim-devicons</li>
    <li>vimPlugins.vim-gh-line</li>
    <li>vimPlugins.vim-go</li>
    <li>vimPlugins.vim-kitty-navigator</li>
    <li>vimPlugins.vim-oscyank</li>
    <li>vimPlugins.vim-ruby</li>
    <li>vimPlugins.vim-snippets</li>
    <li>vimPlugins.vim-tpipeline</li>
    <li>vimPlugins.vim-ultest</li>
    <li>vimPlugins.vimtex</li>
    <li>vimPlugins.vis</li>
    <li>vimPlugins.wilder-nvim</li>
  </ul>
</details>

**Review by figsoda — APPROVED** [review]
I don't have push access but lgtm

**Pull request merged**
